templates/client-daemonset.yaml

```diff
@@ -166,17 +207,17 @@ spec:
             - name: aclconfig
               mountPath: /consul/aclconfig
             {{- end }}
-          lifecycle:
```

Comment:
not something that needs to be changed but we might be adding this back as `consul force-leave -prune` due to https://github.com/hashicorp/consul/issues/6897#issuecomment-571153009